lib/processors/bundlers/moduleBundler.js

```diff
@@ -132,11 +132,12 @@ const log = getLogger("builder:processors:bundlers:moduleBundler");
 				bundle definition
  * @param {module:@ui5/builder/processors/bundlers/moduleBundler~ModuleBundleOptions} [parameters.options.bundleOptions] Module
 				bundle options
+ * @param {string} [parameters.options.targetUi5Version] Optional semver compliant UI5 version e.g '2.0.0' to bundle for. This allows the bundler to make assumptions on available runtime APIs. Omit if the ultimate UI5 version at runtime is unknown or can't be determined.
```

Comment:
Should we make it clear in the description that it is the UI5 core library version, not the UI5 version (e.g. SAPUI5 distribution)? Or do we not want to make a difference here and make sure that if we're e.g. adding a new feature in a new version, that it is introduced with the same UI5 core library and SAPUI5 distribution version? Currently, only the patch versions can differ, but in future the versioning strategy might look different..

Reply:
done. renamed.